src/main/java/shadows/apotheosis/mixin/GoldToolsHaveFortuneModuleMixin.java

```diff
@@ -0,0 +1,29 @@
+package shadows.apotheosis.mixin;
+
+import net.minecraft.nbt.ListTag;
+import net.minecraft.network.chat.Component;
+import net.minecraft.world.item.ItemStack;
+import org.spongepowered.asm.mixin.Mixin;
+import org.spongepowered.asm.mixin.Pseudo;
+import org.spongepowered.asm.mixin.injection.At;
+import org.spongepowered.asm.mixin.injection.Inject;
+import org.spongepowered.asm.mixin.injection.callback.CallbackInfo;
+import org.spongepowered.asm.mixin.injection.callback.CallbackInfoReturnable;
+
+import java.util.List;
+
+@Pseudo
+@Mixin(targets = "vazkii.quark.content.tweaks.module.GoldToolsHaveFortuneModule", remap = false)
+public class GoldToolsHaveFortuneModuleMixin {
+
+    @Inject(method = "fakeEnchantmentTooltip", at = @At("HEAD"), cancellable = true)
+    private static void apoth_blockQuarkEnchTooltip1(ItemStack stack, List<Component> components, CallbackInfo ci) {
+        ci.cancel();
+    }
+
+    @Inject(method = "hideSmallerEnchantments", at = @At("HEAD"), cancellable = true)
+    private static void apoth_blockQuarkEnchTooltip2(ItemStack stack, ListTag tag, CallbackInfoReturnable<ListTag> cir) {
+        cir.setReturnValue(tag);
+    }
+
```

Comment:
May want to double check that the signature of these targets is the same for 1.19

Reply:
they do, see https://github.com/VazkiiMods/Quark/blob/release-1.19.2-3.4-418/src/main/java/vazkii/quark/content/tweaks/module/GoldToolsHaveFortuneModule.java

(also tested it in a modpack with quark installed)